src/components/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import axios from 'axios'
+import { useState, useEffect } from "react";
+
+
+import Topics from "./Topics";
+import LatestArticles from "./LatestArticles/LatestArticles";
+
+
+const HomePage = () => {
+
+  const [allArticles, setAllArticles] = useState([])
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    axios
+      .get("https://nc-news-ajy0.onrender.com/api/articles")
```

Comment:
As per Mezz's feedback, this axios request should be extracted into its own file.